src/pages/components/menu/style.mdx

```diff
@@ -0,0 +1,139 @@
+| Focus                  | Menu option | background-color | `$layer`\*        |
+|                        | Menu option | border           | `$focus`          |
+| Focus and hover        | Menu option | background-color | `$layer-hover` \* |
+|                        | Menu option | text-color       | `$text-primary`   |
+|                        | Icon        | svg              | `$icon-primary`   |
+|                        | Menu option | border           | `$focus`          |
+| Danger hover           | Menu option | background-color | `$support-error`  |
+|                        | Menu option | text-color       | `$text-on-color`  |
+|                        | Icon        | svg              | `$icon-on-color`  |
+| Danger hover and focus | Menu option | background-color | `$support-error`  |
+|                        | Menu option | text-color       | `$text-on-color`  |
+|                        | Icon        | svg              | `$icon-on-color`  |
+| Disabled               | Menu option | background-color | `$layer` \*       |
+|                        | Menu option | text-color       | `$text-disabled`  |
+|                        | Icon        | svg              | `$icon-disabled`  |
+
+<Caption>
+  * Denotes a contextual color token that will change values based on the layer
+  it is placed on.
+</Caption>
+
+<Row>
+<Column colLg={8}>
+
+![Menu interactive states](images/style-menu-interactive-states.png)
+
+</Column>
+</Row>
+
+## Typography
+
+| Element          | Font-size (px/rem) | Font-weight   | Type token         |
+| ---------------- | ------------------ | ------------- | ------------------ |
+| Menu option text | 14 / 0.875         | Regular / 400 | `$body-compact-01` |
+
+## Structure
+
+| Element                            | Property                     | px / rem         | Spacing token                |
+| ---------------------------------- | ---------------------------- | ---------------- | ---------------------------- |
+| Icon                               | icon size                    | 16 / 1           | –                            |
+| Option (default)                   | padding-right, padding-left  | 16 / 1           | `$spacing-05`                |
+| Option (selectable but unselected) | padding-right, padding-left  | 16 / 1, 40 / 2.5 | `$spacing-05`, `$spacing-08` |
+| Option (selectable and selected)   | padding-right, padding-left  | 16 / 1           | `$spacing-05`                |
+| Option (next to the divider)       | margin-top, or margin-bottom | 4 / .25          | `$spacing-02`                |
+| Option (first and last)            | margin-top, or margin-bottom | 4 / .25          | `$spacing-02`                |
+| Divider                            | margin-top                   | 1px              | –                            |
+
+<div className="image--fixed">
+
+![Main menu structure](images/style-menu-structure.png)
```

Comment:
Looks like the structure images might be getting scaled down? I will try to figure this out.